screens/ProfileScreen/profile/index.js

```diff
@@ -62,109 +54,51 @@ const Profile = ({navigation}) => {
       setImageUrl(url);
       setArrayImage([
         ...arrayImages,
-        { id: arrayImages.length + 1, url: url, isChecked: false },
+        {id: arrayImages.length + 1, url: url, isChecked: false},
       ]);
     });
   };
 
   const accessCamera = async () => await requestCameraPermission(selectFile);
 
   useEffect(() => {
-    setModalVisible(false);
-    if (isAnswer === 'PHONE') {
-      accessCamera();
-    } else if (isAnswer === 'STORAGE') {
-      navigation.navigate('Images');
-    }
-    return () => setAnswer('');
+    const handleAnswerChange = () => {
+      setModalVisible(false);
+
+      if (isAnswer === 'PHONE') {
```

Comment:
when I see isAnswer I understand it as a boolean variable, but it turned out that it's a string. This is not ok

Reply:
I will think of a better solution.